nativelib/src/main/scala/scala/scalanative/runtime/ClibFunctions.scala

```diff
@@ -0,0 +1,15 @@
+package scala.scalanative
+
+import native._
+
+/**
+ * C stdlib functions needed by nativelib
+ *
+ * Note: clib depends on nativelib.
+ */
+
+@extern
+private [scalanative] object clib_string {
```

Comment:
You can put the binding into https://github.com/scala-native/scala-native/blob/master/nativelib/src/main/scala/scala/scalanative/runtime/Math.scala and rename the file & object to `Intrinsics` instead.